DXMainClient/DXGUI/Multiplayer/GameInformationPanel.cs

```diff
@@ -43,7 +43,7 @@ public GameInformationPanel(WindowManager windowManager, MapLoader mapLoader)
 
         private GenericHostedGame game = null;
 
-        private bool disposeTextures = false;
+        private bool disposeTextures = true;
```

Comment:
This just caught my attention, I don't think there should be such a flag, just the no map preview texture should be set as a singleton, and then when disposing the texture should be compared to the no map preview one.

Reply:
The `disposeTextures` pattern is directly copied from `MapPreviewBox` class. In that class, `PreloadMapPreviews` is such a texture that should not be disposed. 

I think we can leave this design as a future work since the client already has such a pattern for a long time?

Reply:
I thought that's a trivial change to do so though, isn't that right?

Reply:
Should be but it might require additional tests. Will trying migrating it after I am available again